tasks/msi.py

```diff
@@ -349,7 +392,19 @@ def validate_msi_createfolder_table(db):
     TODO: We don't want the AI flag to be removed from the directories in the allow list either, but
           this behavior was also present in the original installer so leave them for now.
     """
-    allowlist = ["APPLICATIONDATADIRECTORY", "EXAMPLECONFSLOCATION", "checks.d", "run", "logs", "ProgramMenuDatadog"]
+    allowlist = [
```

Comment:
we should make this allowlist per installer. The set of dirs are disjoint I think.